### ISSUE TITLE
false report of `helpful-string` on deleted variable

### ISSUE THREAD

**Issue by KotlinIsland** [KotlinIsland/basedmypy#871]
```py
a = 1

del a

print(f"{a}")  # error: helpful-string
```
